src/components/AddItemForm.tsx

```diff
@@ -1,22 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+interface ListStateType {
```

Comment:
Por padrao, quando estamos criando interfaces de props, elas começam com o nome do componente e terminam com o sufixo `Props`.
```suggestion
interface AddItemFormProps {
```